smarts/env/gymnasium/driving_smarts_competition_env.py

```diff
@@ -39,22 +42,24 @@
     Waypoints,
 )
 from smarts.core.controllers import ActionSpaceType
-from smarts.env.hiway_env import HiWayEnv
-from smarts.zoo.agent_spec import AgentSpec
+from smarts.env.gymnasium.hiway_env_v1 import HiWayEnvV1, SumoOptions
 
 logger = logging.getLogger(__file__)
 logger.setLevel(logging.WARNING)
 
+SUPPORTED_ACTION_TYPES = (ActionSpaceType.RelativeTargetPose,)
```

Comment:
```suggestion
SUPPORTED_ACTION_TYPES = (ActionSpaceType.RelativeTargetPose, ActionSpaceType.TargetPose)
```

Reply:
This issue was hidden because the check for supported action spaces came after actually applying the action space based on the given action type.